.golangci.yaml

```diff
@@ -0,0 +1,44 @@
+linters:
```

Comment:
The linter should be added in a separate PR because we'll need to update the GH Actions as well